### PR TITLE
DER controller

### PR DESCRIPTION
**Pull request by SteffenMeinecke**

### CODE REVIEW

**Comment by vogt31337** [comment]
@SteffenMeinecke please provide a description and maybe rebase your branch since many of the tests aren't running.

**Comment by SteffenMeinecke** [comment]
@vogt31337 @pawellytaev @rbolgaryn I have no idea why this tutorials fail. They should not be affected by the changes of this PR.
FAILED tutorials/plotly_traces.ipynb
FAILED tutorials/plotly_built-in.ipynb
FAILED tutorials/plotly_maps.ipynb

**Comment by SteffenMeinecke** [comment]
The remaining failing tests are not caused by this PR, see failings tests at the develop branch...

File: pandapower/control/controller/pq_control.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+try:
+    from pandaplan.core import pplog
+except:
+    import logging as pplog
+import numpy as np
+
+logger = pplog.getLogger(__name__)
+
+
+class PQController(ConstControl):
```
Comment (pawellytaev) [inline pandapower/control/controller/pq_control.py:16]: is there a good reason to inherit from const_control?

Comment (SteffenMeinecke) [reply pandapower/control/controller/pq_control.py:16]: yes, none of the functions defined in ConstControl are redefined/overwritten so without, the code wouldn't work.

File: tutorials/DER_control_tutorial.ipynb
[the remark this answers is not in the document]
Comment (SteffenMeinecke) [reply tutorials/DER_control_tutorial.ipynb:1]: done

File: pandapower/control/controller/pq_control.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+        **element** - element table ('sgen', 'load' etc.)
+
+        **max_p_error** (float, 0.0001) - Maximum error of active power
+
+        **max_q_error** (float, 0.0001) - Maximum error of reactive power
+
+        **p_ac** (float, 1.0) - Simultaneity factor applied to P and Q
+
+        **f_sizing** (float, 1.0) - Sizing of the converter factor limiting P
```
[the remark this answers is not in the document]
Comment (SteffenMeinecke) [reply pandapower/control/controller/pq_control.py]: I agree

File: pandapower/control/controller/pq_control.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+    OPTIONAL:
+
+        **element** - element table ('sgen', 'load' etc.)
+
+        **max_p_error** (float, 0.0001) - Maximum error of active power
+
+        **max_q_error** (float, 0.0001) - Maximum error of reactive power
+
+        **p_ac** (float, 1.0) - Simultaneity factor applied to P and Q
```
[the remark this answers is not in the document]
Comment (SteffenMeinecke) [reply pandapower/control/controller/pq_control.py]: I agree

**Review by pawellytaev — APPROVED** [review]

**Pull request merged**